README.md

```diff
@@ -141,4 +141,11 @@ Since `Commanding` is not implemented, specifying `AutoEvents` in the configurat
 3. Alternatively the device service can be built natively:
 
         make build
+### Running the device-rest Service as a Snap
+EdgeX Device REST is also available as a snap package. Install the snap with the following command:
+```
+sudo snap install edgex-device-rest
+```
+for more details on the Device REST Snap, including installation, configuration, please refer to [EdgeX REST Device Service Snap](https://github.com/edgexfoundry/device-rest-go/tree/main/snap)
 
+for more details on Snap, including EdgeX Snap, viewing logs, security services, please check [Getting Started with Snap](https://docs.edgexfoundry.org/2.0/getting-started/Ch-GettingStartedSnapUsers/)
```

Comment:
2x ~for~ For 

more details on ~Snap, including EdgeX Snap~ deploying EdgeX with Snaps, viewing logs

Reply:
Updated. Thanks!